pages/user/resume.js

```diff
@@ -27,7 +27,7 @@ const ResumePage = () => {
           <div className="w-3/5 flex flex-col flex-grow">
             <ResumeUpload resume={session?.user?.resume} />
             <PDFViewer pdf={session?.user?.resume} />
```

Comment:
dont need the session?.user?.resume protected page handles it

Reply:
![image](https://github.com/acm-ucr/membership-portal/assets/53993828/98d8a305-fe4e-4072-a9ee-6048ef2483b1)
errors when ternary operator is removed